tests/components/google_assistant/test_google_assistant.py

```diff
@@ -360,7 +440,7 @@ async def test_execute_request(hass_fixture, assistant_client, auth_header):
     body = await result.json()
     assert body.get("requestId") == reqid
     commands = body["payload"]["commands"]
-    assert len(commands) == 6
+    assert len(commands) == 9
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)